core/server/src/main/java/alluxio/worker/SessionCleaner.java

```diff
@@ -29,7 +28,7 @@
 public final class SessionCleaner implements Runnable {
```

Comment:
This is probably outside the scope of this PR, but could this class be replaced with HeartbeatThread + SleepingTimer?